tests/access/test_AccessControl.py

```diff
@@ -0,0 +1,178 @@
+import pytest
+from pathlib import Path
+from starkware.starknet.testing.starknet import Starknet
+from utils import (
+    TestSigner, TRUE, FALSE,
+    assert_event_emitted, assert_revert,
+    get_contract_def, cached_contract
+)
+
+DEFAULT_ADMIN_ROLE = 0
+SOME_OTHER_ROLE = 42
```

Comment:
what do you think about suggesting a way to compute these roles? i'm sure many people will copy whatever we do here

Reply:
I should also add my new finding that felts can indeed hold 252 bits, BUT a defined `constant` can only hold 251 bits

Reply:
!!

Reply:
That's not true. A felt can hold at max [`2**251 + 17 * 2**192 + 1`](https://github.com/starkware-libs/cairo-lang/blob/167b28bcd940fd25ea3816204fa882a0b0a49603/src/starkware/cairo/lang/cairo_constants.py#L1) which is greater than `2**251` but less than `2**252`.

Reply:
> That's not true. A felt can hold at max [`2**251 + 17 * 2**192 + 1`](https://github.com/starkware-libs/cairo-lang/blob/167b28bcd940fd25ea3816204fa882a0b0a49603/src/starkware/cairo/lang/cairo_constants.py#L1)-1 which is greater than `2**251` but less than `2**252`.

Thank you for the correction^. To rephrase, since a defined `constant` can hold a max between `2**251` and `2**252`, these felts can store a guaranteed 251 bits